ci/vcpkg/x64-osx-static-debug.cmake

```diff
@@ -21,5 +21,7 @@ set(VCPKG_LIBRARY_LINKAGE static)
 
 set(VCPKG_CMAKE_SYSTEM_NAME Darwin)
 set(VCPKG_OSX_ARCHITECTURES x86_64)
+set(VCPKG_OSX_DEPLOYMENT_TARGET "10.13")
+set(VCPKG_OSX_SYSROOT "/Library/Developer/CommandLineTools/SDKs/MacOSX10.13.sdk")
```

Comment:
Could you add this to `-release.cmake` too?

Reply:
Ahh, build log is here https://github.com/ursacomputing/crossbow/runs/2900482044

Reply:
Still bot `-isysroot` paths get passed.